_services/fractional-coo.html

```diff
@@ -3,5 +3,6 @@
 name: fractional-coo
 title: Fractional COO
 icon: "/img/Fractional.png"
+alt: "glowing pizza slice fitting into the missing space on a pizza, symbolizing how deranged’s staff seamlessly integrate into your business as executives"
```

Comment:
The part before "symbolizing" is good. Only describe what is on the image, leave subtext as subtext in alt text too.